src/ui/libs/DatalensChartkit/ChartKit/plugins/Table/renderer/components/Table/usePreparedTableData.ts

```diff
@@ -191,12 +192,12 @@ export const usePreparedTableData = (props: {
     const rowMeasures = React.useRef<Record<string, number>>({});
     React.useEffect(() => {
         rowMeasures.current = {};
-    }, [data, cellSizes]);
+    }, [data, dimensions]);
 
     const rowVirtualizer = useVirtualizer({
         count: tableRows.length,
         estimateSize: () => {
-            return 30;
+            return 60;
```

Comment:
Why exactly 60? :)

Reply:
By recommendation at Tanstack Virtual: "If you are dynamically measuring your elements, it's recommended to estimate the largest possible size"
60 - it is approximately row with one line break